shell.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import ./nix { } }:
+
+let
+  nodePackages = with pkgs.nodePackages; [
+    typescript
+    typescript-language-server
+    purescript-language-server
+    prettier
+    yaml-language-server
+    vscode-html-languageserver-bin
```

Comment:
Maybe add `vscode-json-languageserver-bin` for JSON and `dhall-lsp-server`.

Reply:
Sure.